terraform/modules/hub/modules/ecs_fargate_app/route53.tf

```diff
@@ -0,0 +1,23 @@
+resource "aws_route53_zone" "cluster" {
```

Comment:
Is this needed?

Reply:
Maybe. Everything in this module is a copy of the equivalent from ecs_app, and unless there's a Fargate-specific reason to change it I'd prefer to keep it the same for now.